src/utilities.js

```diff
@@ -250,3 +251,92 @@ export function getResults(request) {
     });
   };
 }
+
+export function getTrendingResults(request) {
+  return (searchType, obj) => {
+    const map = {
+      'Daily trends': {
+        path: '/trends/api/dailytrends',
+        extraParams: {
+          ed: convertDateToString(obj.trendDate, false, true),
+          ns: obj.ns,
+        },
+      },
+      'Real time trends': {
+        path: '/trends/api/realtimetrends',
+        extraParams: {
+          fi: 0,
+          fs: 0,
+          ri: 300, // # of trending stories IDs returned
+          rs: 20,
+          sort: 0,
+        },
+      },
+    };
+
+    let options = {
+      method: 'GET',
+      host: 'trends.google.com',
+      path: map[searchType].path,
+      qs: {
+        hl: obj.hl,
+        tz: obj.timezone,
+        geo: obj.geo,
+        cat: obj.category,
+      },
+    };
+
+    if (obj.agent) options.agent = obj.agent;
+
+    Object.assign(options.qs, options.qs, map[searchType].extraParams);
+
+    return request(options)
+    .then((res) => {
+      try {
+        /** JSON.parse will decode unicode */
+        const results = JSON.stringify(JSON.parse(res.slice(5)));
```

Comment:
No need to assign variable `results` since we immediately return it:
```
try {
        /** JSON.parse will decode unicode */
        return JSON.stringify(JSON.parse(res.slice(5)));
```

Reply:
agreed